system/mrm_handler/src/mrm_handler/mrm_handler_core.cpp

```diff
@@ -115,12 +115,11 @@ void MrmHandler::publishHazardCmd()
   HazardLightsCommand msg;
 
   msg.stamp = this->now();
-  if (is_emergency_holding_) {
-    // turn hazard on during emergency holding
-    msg.command = HazardLightsCommand::ENABLE;
-  } else if (isEmergency() && param_.turning_hazard_on.emergency) {
-    // turn hazard on if vehicle is in emergency state and
-    // turning hazard on if emergency flag is true
+  if (
```

Comment:
@mkuri @TetsuKawa 
The original implementation looks wierd: Is the following correct?

```
if (param_.turning_hazard_on.emergency) {
  if (is_emergency_holding_ || isEmergency()) {
    msg.command = HazardLightsCommand::ENABLE;
  } else {
    msg.command = HazardLightsCommand::NO_COMMAND;
  }
} else {
  msg.command = HazardLightsCommand::NO_COMMAND;
}
```